nsxt/resource_nsxt_lb_pool.go

```diff
@@ -134,6 +138,38 @@ func getPoolMembersSchema() *schema.Schema {
 					Type:        schema.TypeInt,
 					Description: "Pool member weight is used for WEIGHTED_ROUND_ROBIN balancing algorithm. The weight value would be ignored in other algorithms",
 					Optional:    true,
+					Default:     1,
+				},
+			},
+		},
+	}
+}
+
+func getPoolMemberGroupSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Description: "Dynamic pool members for the loadbalancing pool. When member group is defined, members setting should not be specified",
+		Optional:    true,
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"grouping_object": getSingleResourceReferencesSchema(true, false, []string{"NSGroup"}, "Load balancer pool support grouping object as dynamic pool members. The IP list of the grouping object such as NSGroup would be used as pool member IP setting"),
+				"ip_revision_filter": &schema.Schema{
```

Comment:
Is it just me or is it weird naming in NSX? Shouldn't this be called ip_version_filter?

Reply:
yes... I'll change the name in our api.